### PR TITLE
JumpIconMotion and LaunchMotion icons added

### PR DESCRIPTION
**Pull request by jagathgj**
contributed to #16 

image

### CODE REVIEW

**Review by kristastarr — COMMENTED** [review]
The code looks good, and I really like the animations as well! 🚀 I know you and @johnbister have been discussing the animations - @johnbister do you have any feedback?

File: src/components/Operations/LaunchMotion/LaunchMotion.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    setIsAnimating(props.isAnimating)
+  }, [props.isAnimating])
+
+  const shouldAnimate = isAnimating ? styles.isAnimating : ''
+
+  return (
+    <div ref={iconRef} className={`${shouldAnimate}`}>
+      <svg style={iconStyles} viewBox='0 0 32 32' className={styles.LaunchMotion}>
+        <title>Launch</title>
+        <path  d="M26,28H6a2.0027,2.0027,0,0,1-2-2V6A2.0027,2.0027,0,0,1,6,4H16V6H6V26H26V16h2V26A2.0027,2.0027,0,0,1,26,28Z"/>
```
Comment (johnbister) [inline src/components/Operations/LaunchMotion/LaunchMotion.js:23]:
```suggestion
        <path  d="M24.6,6L16,14.6l1.4,1.4L26,7.4L24.6,6z M24,30H4c-1.1,0-2-0.9-2-2V8c0-1.1,0.9-2,2-2h10v2H4v20h20V18h2v10
		C26,29.1,25.1,30,24,30z"/>
```

File: src/components/Operations/LaunchMotion/LaunchMotion.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <polygon className={styles.LaunchTip}  points="20 2 20 4 26.586 4 18 12.586 19.414 14 28 5.414 28 12 30 12 30 2 20 2"/>
```
Comment (johnbister) [inline src/components/Operations/LaunchMotion/LaunchMotion.js:24]:
```suggestion
      <path className={styles.LaunchTip} d="M18,4v2h6.6L16,14.6l1.4,1.4L26,7.4V14h2V4H18z" />
```

File: src/components/Operations/LaunchMotion/launch--motion.module.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+@keyframes launch {
+  0% {
+    transform: translate(0px, 0px);
+  }
+  10% {
+    transform: translate(-2px, 2px);
```
Comment (johnbister) [inline src/components/Operations/LaunchMotion/launch--motion.module.scss:6]:
```suggestion
    transform: translate(3px, -3px);
```

File: src/components/Operations/LaunchMotion/LaunchMotion.js (hunk 1 above)
Comment (johnbister) [reply src/components/Operations/LaunchMotion/LaunchMotion.js]: Jumplink looks good to go! Great job! The original launch file was a little cramped so I edited it to allow enough room for the arrow to go forward. I know we originally talked about having a pullback before going forward, but I think it makes more sense in terms of consistency to go with the arrow just moving forward.  Let me know if you are OK with these little tweaks for launch. Overall, they look great!

**Review by johnbister — COMMENTED** [review]
Jumplink is perfect! Great job! Let me know what you think of the edits I suggested for launch.

**Comment by johnbister** [comment]
@jagathgj Looks awesome! Jumplink is perfect--good to go. I noticed there wasn't enough room in the original svg for 'launch' to have the arrow move forward so I suggested a couple of edits in a commit. If you're comfortable with those tweaks, let me know and we can merge this in :smile: Thank you again!

**Comment by jagathgj** [comment]
Hey @johnbister, The launch icon now looks fantastic. I believe you divided the svg into two sections.
Please merge the changes.

**Pull request merged**